website/docs/patterns/table-multi-select/partials/interaction-concepts/interaction-concepts.md

```diff
@@ -0,0 +1,57 @@
+When combining a Multi-select table with filtering, sorting, or pagination, you may encounter complex scenarios regarding how selected results should be persisted across pages and what the user expects when traversing large data sets.
+
+These interaction guidelines are not fully comprehensive. They should instead be considered as conversation starters between design and engineering on how best to optimize the user experience.
+
+!!! Info
+
+If you run into unique scenarios that these guidelines don’t account for, we would love to hear about them through our normal [support](/about/support) channels.
+!!!
+
+## Consistent communication
+
+The most important method to ensure users are interacting with the results they intend to is to clearly communicate what they have selected, or even that a selection of results exists, **at all times**.
+
+This can help to prevent unnecessary deletions of results or making unintended changes if what is currently selected is unclear.
```

Comment:
```suggestion
This can help to prevent unintended deletion of data or unintended changes.
```

Reply:
These two sentences could probably still use a bit of massaging to further clarify but I attempted to simplify them a bit...

Reply:
I've done a bit to combine these sentences and add a bit of clarifying imagery and information.